docs/sources/tempo/configuration/_index.md

```diff
@@ -640,7 +640,10 @@ query_frontend:
 
         # The number of shards to break ingester queries into.
         [ingester_shards]: <int> | default = 3]
-        
+
+        # The maximum allowed value of spans per span set. 0 disables this limit.
+        [max_spans_per_span_set]: <int> | default = 100]
```

Comment:
Should we set it to 0 by default? then it won't be a breaking change

Reply:
That's okay to me. It simplify setting up Tempo for sure

Reply:
I think this needs more context so people know what a limit of 100 spansets mean. Are we talking about results of a traceql query?

Reply:
this is a limit of 100 spans per spanset. it would be this field in Grafana:

![image](https://github.com/user-attachments/assets/829f5dc7-f763-4914-a089-6fa607f1689a)

naturally this can be adjusted to whatever works for your org (or turned off with 0)